src/UI/Factory.php

```diff
@@ -662,4 +662,36 @@ public function breadcrumbs(array $crumbs);
 	 */
 	public function chart();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     A Floatable overlays all other content on the screen. It's free floating
+	 *     nature allows the positioning to some fitting space on the screen. They may
+	 *     persist during navigational changes allowing to cary information through
+	 *     different screens.
+	 *   composition: >
+	 *      Floatables consist of a Title and a glyphed close-button in a header and a
+	 *      content area placed bellow which may offer actions.
```

Comment:
The actions come from the exercise assignment that "lauch" the instructions. Examples would be: Hand-in Text, Hand-in Portfolio, Hand-in Blog